automatilib/cola/views.py

```diff
@@ -64,7 +64,7 @@ def get(self, request: HttpRequest, **kwargs: dict) -> HttpResponse:
 
         redirect_url = settings.LOGIN_URL
         response = redirect(reverse(redirect_url))
-        response.delete_cookie(settings.COLA_COOKIE_NAME)
+        response.delete_cookie(settings.COLA_COOKIE_NAME, domain=".cabinetoffice.gov.uk")
         return response
```

Comment:
Could we move the `logout` to after the cookie has been deleted. If something goes wrong they could be logged out with a valid cookie still

Reply:
good idea [9cd314c](https://github.com/i-dot-ai/automatilib/pull/27/commits/9cd314c5fe8948e2ea5fc874e9f12d2f21989cbd)